code/pages/econsent/tasks.html

```diff
@@ -33,15 +33,18 @@ <h3>{{@month}} {{@day}}</h3>
         </div>
 
         <div data-for="@tasks.item">
-            <div class="button-group-wrapper">
-                <button class="general-button" data-tag="@tag" type="button">
-                    {{@task}}
-                </button>
+            <div data-if="@showTask">
+                <div slot="true">
```

Comment:
This div with slot true is not necessary if you don't have a slot false. You can remove it.